composer.json

```diff
@@ -12,7 +12,7 @@
         }
     ],
     "require": {
-        "friendsofphp/php-cs-fixer": "~2.8.0"
+        "friendsofphp/php-cs-fixer": "~2.9"
```

Comment:
We had some breaking changes with php-cs-fixer v2 previously (https://github.com/M6Web/php-cs-fixer-config/pull/8/files), that's why we decided to remove the composer semver notation and stick to 2.8 only as php-cs-fixer didn't seem to be semver ready.
We should ensure that upgrading to 2.9 doesn't break anything on our internal repositories before doing that. In all cases we should better stick to `~2.9.0` instead of `~2.9`.